### ISSUE TITLE
"Explain (AI)" Does Not Correctly Collapse to Next Collapsed Item

### ISSUE THREAD

**Issue by Talmanes** [gitkraken/vscode-gitlens#3645]
## Problem

When collapsing the **Explain (AI)** item, it doesn't collapse to the next item, in this case **Line History**.  Did not find a duplicate of this in this repo or VSCode.

![Image](https://github.com/user-attachments/assets/56443eb1-037b-4c17-93cd-77877183f322)

## Expected

I would expect the window to collapse between **Explain (AI)** and **Line History**.  Any extra empty space I would expect to see below **Search & Compare**.

### GitLens Version

15.5.1

### VS Code Version

Version: 1.94.0 (system setup)
Commit: d78a74bcdfad14d5d3b1b782f87255d802b57511
Date: 2024-10-02T13:08:12.626Z
Electron: 30.5.1
ElectronBuildId: 10262041
Chromium: 124.0.6367.243
Node.js: 20.16.0
V8: 12.4.254.20-electron.0
OS: Windows_NT x64 10.0.22631

**Comment by sergeibbb** [comment]
@d13 

I don't see this view anymore. Have we moved it?

**Comment by eamodio** [comment]
Yeah, that panel has been removed in favor of the Explain button at the top. 

Technically this still applies to the Files panel, as those panels are within our webview and not separate vscode views. But extensions have no control over the relative sizing of the view they are contained within, so there isnt much if anything that can be done here.

**Comment by sergeibbb** [comment]
@eamodio 

Maybe if we could increase "Inspect: commit details" section and put the collapsed sub-panels to the bottom it would look more natural?

<img width="400" src="https://github.com/user-attachments/assets/99884108-f41f-4ee4-b2ec-88f31e7c48ee">

**Comment by eamodio** [comment]
We will likely not allow the Files Changed section to even be collapsable anymore, so this won't be an issue at all.

**Comment by sergeibbb** [comment]
@eamodio Then I'm closing as "won't fix". cc @d13
